test/util/testnode/node_interaction_api.go

```diff
@@ -50,6 +52,10 @@ func NewContext(goContext context.Context, keyring keyring.Keyring, tmConfig *tm
 	return Context{goContext: goContext, Context: clientContext, apiAddress: apiAddress}
 }
 
+func (c *Context) GetTMNode() *node.Node {
+	return c.tmNode
+}
+
```

Comment:
Why was this added? It doesn't look used

Reply:
not sure, removed in 6c70dac7ce849046c8e7dbbb32f68614ef523cb9